plugin-colorpicker/colorpicker.cpp

```diff
@@ -56,7 +56,7 @@ ColorPickerWidget::ColorPickerWidget(QWidget *parent):
     QFrame(parent)
 {
     QFontMetrics fm (mLineEdit.font());
-    mLineEdit.setFixedWidth ( 10*fm.width (QStringLiteral("a")) );
+    mLineEdit.setFixedWidth ( 10*fm.horizontalAdvance (QStringLiteral("a")) );
```

Comment:
I think it's the time for bumping `REQUIRED_QT_VERSION` to the last LTS, 5.12. `QFontMetrics::horizontalAdvance()` doesn't exist in 5.10.

Reply:
Damn it... :(

Reply:
IMHO, this is good but 5.10 is too old.

Reply:
OK. Bumped the required Qt version.